inst/sql/sql_server/field_plausible_start_before_end.sql

```diff
@@ -0,0 +1,61 @@
+
+/*********
+PLAUSIBLE_START_BEFORE_END
+all start dates are before their corresponding end dates (PLAUSIBLE_START_BEFORE_END == Yes).
+@cdmFieldName is the start date and @plausibleStartBeforeEndFieldName is the end date.
+
+Parameters used in this template:
+schema = @schema
+cdmTableName = @cdmTableName
+cdmFieldName = @cdmFieldName
+plausibleTemporalAfterTableName = @plausibleTemporalAfterTableName
+plausibleTemporalAfterFieldName = @plausibleTemporalAfterFieldName
```

Comment:
lines 11 and 12 are duplicated, and should reference `@plausibleStartBeforeEndFieldName` instead